packages/lib-user/src/utils/createPanoptesMembership.js

```diff
@@ -1,17 +1,19 @@
 import { panoptes } from '@zooniverse/panoptes-js'
 import auth from 'panoptes-client/lib/auth'
 
-export async function createPanoptesMembership({
-  groupId,
-  joinToken,
-  userId
+export async function createPanoptesMembership(key, {
+  arg: {
+    groupId,
+    joinToken,
+    userId
+  }
 }) {
   const token = await auth.checkBearerToken()
   const authorization = `Bearer ${token}`
   if (!token) return null
 
   try {
-    const response = await panoptes.post('/memberships',
+    const { body } = await panoptes.post('/memberships',
```

Comment:
Any chance of `!response.ok` and no `body` from a POST request here?

Reply:
Yes I think so. I've added some sad path handling.